From 872cd480990e6536d2cf48568627903ebf5acbeb Mon Sep 17 00:00:00 2001
From: Dan Hor <dan@danny.cz>
Date: Tue, 24 Oct 2023 10:36:09 +0200
Subject: [PATCH] move from distutils to sysconfig

The distutils module was deprecated in Python 3.10, thus switch to the
sysconfig module instead.

Upstream: committed abd19b721f7e9b4d514ed319ece173ebc7b1ea72

Signed-off-by: Dan Hor <dan@danny.cz>
Signed-off-by: Adam Duskett <adam.duskett@amarulasolutions.com>
---
 python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 5b6f420..19ce500 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -42,7 +42,7 @@ endif ()
 
 set_target_properties ( ${SWIG_MODULE_ftdi1_REAL_NAME} PROPERTIES NO_SONAME ON )
 
-execute_process ( COMMAND ${PYTHON_EXECUTABLE} -c "from distutils import sysconfig; print( sysconfig.get_python_lib( plat_specific=True, prefix='${CMAKE_INSTALL_PREFIX}' ) )"
+execute_process ( COMMAND ${PYTHON_EXECUTABLE} -c "import sysconfig; print( sysconfig.get_path( 'platlib', vars={'platbase': '${CMAKE_INSTALL_PREFIX}'} ) )"
                   OUTPUT_VARIABLE _ABS_PYTHON_MODULE_PATH
                   OUTPUT_STRIP_TRAILING_WHITESPACE )
 
-- 
2.41.0

